code/modules/uplink/uplink_item.dm

```diff
@@ -1026,6 +1026,49 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 		else
 			return ..()
 
+/datum/uplink_item/cyber_implants/esword	//This is a bad idea
+	name = "Arm Energy-Blade Projector"
+	desc = "An illegal cybernetic combat implant that allows the user to project a blade of lethal energy from their hand."
+	item = /obj/item/device/autoimplanter/esword
+	cost = 12
+	player_minimum = 30
+
+/datum/uplink_item/cyber_implants/esword/traitor	//This is a worse idea
+	name = "Arm Energy-Blade Projector"
+	desc = "An illegal cybernetic combat implant that allows the user to project a blade of lethal energy from their hand."
+	item = /obj/item/device/autoimplanter/esword
+	cost = 16
+	include_modes = list()
+	exclude_modes = list(/datum/game_mode/nuclear)
+	player_minimum = 30
+
+/datum/uplink_item/cyber_implants/medbeam
+	name = "Medical Beam Arm Implant"
+	desc = "An cybernetic implant that gives the user an extendable medical beamgun in their arm"
+	item = /obj/item/device/autoimplanter/medbeam
```

Comment:
Admins can spawn their own implanters
Just do the change. Doing it this way is horribly inneficient